src/Microsoft.Health.Dicom.SqlServer/Features/Schema/Sql/Tables/Instance.sql

```diff
@@ -157,3 +157,19 @@ INCLUDE
     OriginalWatermark
 )
 WITH (DATA_COMPRESSION = PAGE)
+
+-- Used in migrating files
+CREATE NONCLUSTERED INDEX IX_Instance_Watermark_Status_CreatedDate on dbo.Instance
```

Comment:
What does the query plan look like with this index?

Reply:
![image](https://user-images.githubusercontent.com/3883800/236026025-714d98c1-6797-46fb-801e-810331308712.png)